lib/modules/manager/gradle/extract/consistent-versions-plugin.ts

```diff
@@ -0,0 +1,187 @@
+import { logger } from '../../../../logger';
+import { newlineRegex, regEx } from '../../../../util/regex';
+import type { PackageDependency } from '../../types';
+import type { GradleManagerData } from '../types';
+
+export const VERSIONS_PROPS = 'versions.props';
+export const VERSIONS_LOCK = 'versions.lock';
+
+/**
+ * Determines if Palantir Gradle consistent-versions is in use, https://github.com/palantir/gradle-consistent-versions.
+ * The plugin name must be in build file and both `versions.props` and `versions.lock` must exist.
+ *
+ * @param availableFiles list of gradle build files found in project
+ * @param fileContents map with file contents of all files
+ */
+export function usesGcv(
+  availableFiles: string[],
+  fileContents: Record<string, string | null>
+): boolean {
+  if (
+    fileContents['build.gradle']?.includes(
+      'com.palantir.consistent-versions'
+    ) ||
+    fileContents['build.kts']?.includes('com.palantir.consistent-versions')
```

Comment:
I guess you mean `build.gradle.kts` instead?

Reply:
Fixed.